docs/AddingCustomOp.md

```diff
@@ -8,6 +8,10 @@ Note: These APIs are experimental and will change in the next release. They're r
 * Create an OrtCustomOp structure for each op and add them to the OrtCustomOpDomain with OrtCustomOpDomain_Add
 * Call OrtAddCustomOpDomain to add the custom domain of ops to the session options
 See [this](../onnxruntime/test/shared_lib/test_inference.cc) for an example called MyCustomOp that uses the C++ helper API (onnxruntime_cxx_api.h).
+You can also compile the custom ops into a shared library and use that to run a model via the C++ API. The same test file contains an example.
```

Comment:
C/C++ APIs
Can also remove the 'experimental' word from the text. It's a public API and no longer experimental.